docs/go-vanilla/config.yaml

```diff
@@ -0,0 +1,16 @@
+name: Golang (vanilla)
```

Comment:
My take is that we can make the folder name just `go`, and the name "Go" for vanilla since the other higher level abstractions all have names. Kind of like `python` vs `flask` or `django`.